app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class UsersController < ApplicationController
+  around_action :unscope_resources
+  load_resource :user
+
+  def show
+    @course_users = @user.course_users.with_course_statistics
+    @current_course_users = @course_users.from_instance(current_tenant).merge(Course.current)
+    @completed_course_users = @course_users.from_instance(current_tenant).merge(Course.completed)
+    @instances = other_instances
+  end
+
+  private
+
+  def unscope_resources
+    Course.unscoped do
+      yield
+    end
+  end
+
+  def other_instances
+    @user.course_users.map(&:course).map(&:instance).uniq - [current_tenant]
```

Comment:
You can get the instances through: `ActsAsTenant.without_tenant { @user.instance_users.map(&:instance) }`, I think we should show all instances instead.